SU-SOE/engineering_profile
# SOED9-109 - pulled files form SOED9-91 to skirt having to merge.

# READY FOR REVIEW

# Summary
- This PR contains the config files necessary to migrate SOE news from its D7 site to the new SOE D9 site. 

# Needed By (Date)
ASAP

# Urgency
- High

# Steps to Test

1. Fetch this branch into your local and check it out.
2. Log into the site and go to `/admin/content`. Once there, remove all nodes with a 'news' content type. 
3. Run the command to import migration configs: run `drush cim -y` to import the new configs.
4. Run `drush migrate:import --group=news_items_d7 --limit=20`
5. Go back to `/admin/content' and open a few of the newly imported news items. And check for anything that might be missing, such as images, bylines, deks, etc...


# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOED9-109
- https://stanfordits.atlassian.net/browse/SOED9-91

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
